bedrock/base/templates/includes/site-footer.html

```diff
@@ -17,6 +17,9 @@
           <li><a href="{{ url('mozorg.contact.contact-landing') }}" data-link-type="footer" data-link-name="Contact Us">{{ _('Contact Us') }}</a></li>
           <li><a href="{{ donate_url('footer') }}" class="donate" data-link-type="footer" data-link-name="Donate">{{ _('Donate') }}</a></li>
           <li><a href="https://wiki.mozilla.org/Webdev/GetInvolved/mozilla.org" data-link-type="footer" data-link-name="Contribute to this site">{{ _('Contribute to this site') }}</a></li>
+          {% if template_source_url %}
+            <li><a href="{{ template_source_url }}" data-link-type="footer" data-link-name="Edit this page">{{ _('Edit this page') }}</a></li>
```

Comment:
This adds a new string to the shared footer, so we'll either need to show it for `en` only temporarily, or let this sit while the new string is translated?